conans/client/tools/scm.py

```diff
@@ -343,10 +344,9 @@ def is_pristine(self):
                         return False
                 return True
         else:
-            import warnings
-            warnings.warn("SVN::is_pristine for SVN v{} (less than {}) is not implemented, it is"
-                          " returning not-pristine always because it cannot compare with"
-                          " checked out version.".format(self.version, SVN.API_CHANGE_VERSION))
+            logger.warning("SVN::is_pristine for SVN v{} (less than {}) is not implemented, it is"
```

Comment:
I am not sure of this change, as logger.warning is very rarely activated. I'd say that we would want an ``output.warn()`` instead.